modules/encrypted_cos_bucket/README.md

```diff
@@ -0,0 +1,215 @@
+<!-- Update the title -->
+# encrypted_cos_bucket module
```

Comment:
Use a more human readable title here (the actual module title is already below on line 70)